www/pages/[id].tsx

```diff
@@ -203,8 +203,7 @@ function SnippetComponent(props: {
           </span>
         )}
         <div
-          class={tw
-            `px-4 py-4 text-sm overflow-scroll sm:overflow-hidden relative` +
+          class={tw`px-4 py-4 text-sm overflow-scroll relative` +
```

Comment:
Removing the overflow adds a bunch of scroll bars to each of the snippets. The code would probably require a larger refactor so all of the code blocks scroll in sync, with a single scroll bar at the bottom.

You can see this problem in the deployment preview.

Reply:
I see. Maybe the problem is the symmetric "side-to-side" code highlight system with multiple instances of a highlighter and overflow possibilities...

We can try make something more robust with one highlight code block with dynamic superpowers like [stripe](https://stripe.com/docs/checkout/quickstart) or [stitches](https://stitches.dev/).